spec/apidoc-spec.json

```diff
@@ -122,7 +122,8 @@
 	{ "name": "description", "type": "string", "required": false },
```

Comment:
I think the right design here is one of:

  a. add `discriminator_default` field
  b. allow `discriminator` to be an object that looks like `{ "name": "xxx", "default": "yyy" }`
  c. place a default: true on the specific `type` in the `types` list

I think (c) is the cleanest of all of these